probe.c

```diff
@@ -61,120 +69,161 @@ void probe_init()
 
 void probe_check()
 {
-  if (probe_get_active_sensor_state())
+  if (probe_get_active_sensor_state()) {
```

Comment:
Fair enough.  This brings up another issue though: Why do we have todo's with dead code laying about? If it isn't used, pull it out the commented code and leave a TODO.  Leaving 'maybe some day' statements around just creates code smell.